ccmbioinfo/lfs
# LFS-1200: Cards4Care Enrollment Status Update - Add resumed and paused

- Renames the on_hold participant status to paused
- Adds resumed status
- Adds details sections for both paused and resumed

To test the migrator:
- create one or more Participant Status forms
- set different values for the status question, including a few on Paused that will be manually changed to `on_hold`
- open Composum browser, navigate to the forms that will be switched to `on_hold`, check them out, navigate to the status answer (should be the first answer in the first section), manually change the value to `on_hold`, checkin the form
- open `http://localhost:8080/Questionnaires/Participant%20Status.fixstatus.html`, check that the right forms are listed to be migrated (only the ones manually set to `on_hold`), then confirm the changes
- check that the on hold forms are now paused, and check that all other forms still have their old values
